src/h5p-editor-csv-to-text.ts

```diff
@@ -82,28 +85,39 @@ class CSVToTextWidget extends H5PWidget<H5PFieldText> implements IH5PWidget {
 
   private static createFileInput(
     eventListener: (event: Event) => void,
-  ): HTMLInputElement {
+  ): HTMLDivElement {
     const inputId = H5P.createUUID();
+    
+    const fileInputWrapper = document.createElement("div");
+    fileInputWrapper.classList.add("file");
+    fileInputWrapper.classList.add("csvtotext-file");
+
+    const fileInputLabelText = document.createElement("div");
+    fileInputLabelText.classList.add("h5peditor-field-file-upload-text");
+    fileInputLabelText.innerHTML += "Import CSV-file";
+
     const fileInputElement = document.createElement("input");
     fileInputElement.id = inputId;
     fileInputElement.type = "file";
     fileInputElement.accept = "text/csv";
     fileInputElement.multiple = true;
     fileInputElement.addEventListener("change", eventListener);
+    fileInputElement.classList.add("csvtotext-input");
+
+    const fileInputLabel = document.createElement("label");
+    fileInputLabel.classList.add("add");
+    fileInputLabel.appendChild(fileInputLabelText);
+    fileInputLabel.appendChild(fileInputElement);
 
-    // TODO: Move CSS into a separate CSS file
-    fileInputElement.style.display = "inline-block";
-    fileInputElement.style.cursor = "pointer";
-    fileInputElement.style.padding = "0.5em 1.5em 0.5em 3em";
-    fileInputElement.style.background =
-      "linear-gradient(to bottom, #fbfbfb 0, #f2f2f2 100%)";
-    fileInputElement.style.border = "1px solid #d0d0d1";
-    fileInputElement.style.borderRadius = "0.25em";
-    fileInputElement.style.color = "#222222";
-    fileInputElement.style.fontWeight = "bold";
-    fileInputElement.style.lineHeight = "normal";
-
-    return fileInputElement;
+    const fileInputValue = document.createElement("div");
+    fileInputValue.id = "csv-to-text-value";
```

Comment:
This ID will be duplicated if someone were to use this widget twice in the same semantics schema. Can we add the unique `inputId` to the ID?

If not, instead of searching for `document['csv-to-text-value']` in `insertIntoField`, we could add `fileInputValue` as a parameter in `insertIntoField`.

Reply:
Added it as a parameter now